helm/frost-server/values.yaml

```diff
@@ -276,6 +287,28 @@ frost:
       recvQueueSize: 2000
       maxInFlight: 50
 
+    # FROST-Server MQTT lifecycle options.
+    lifecycle:
+
+    # FROST-Server MQTT volumeMounts options.
+    volumeMounts:
+
+    # FROST-Server MQTT volumes.
+    volumes:
+
+    securityContext:
+
+    ingress:
```

Comment:
Now there are two `ingress` blocks in the mqtt section?

Reply:
That's true. The empty ingress key is useless and must be removed.